### PR TITLE
Add DFT logo and info

### PR DESCRIPTION
**Pull request by DefigoNetwork**
Addin DFT logo and info

### CODE REVIEW

File: blockchains/binance/assets/0x787eCc05073008311CBbC97Bf7fc9C600c4BB226/info.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+ "name": "DFT",
```
Comment (Panda777Igor) [inline blockchains/binance/assets/0x787eCc05073008311CBbC97Bf7fc9C600c4BB226/info.json:2]: "Web3Coin"

File: blockchains/binance/assets/0x787eCc05073008311CBbC97Bf7fc9C600c4BB226/info.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    "website": "https://defigo.network/",
+    "description": "First Cross-chain Defi Platform 3.0",
+    "explorer": "https://bscscan.com/token/0x787ecc05073008311cbbc97bf7fc9c600c4bb226?a=0x2473a3b955ebffdff25079730d63851cd0437485",
+    "type": "BEP20",
+    "symbol": "DFT",
+    "decimals": 18,
+    "status": "active",
+    "id": "0x787eCc05073008311CBbC97Bf7fc9C600c4BB226"
```
Comment (Panda777Igor) [inline blockchains/binance/assets/0x787eCc05073008311CBbC97Bf7fc9C600c4BB226/info.json:10]:
```suggestion
    "id": "0x787eCc05073008311CBbC97Bf7fc9C600c4BB226"
```

File: blockchains/binance/assets/0x787eCc05073008311CBbC97Bf7fc9C600c4BB226/info.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+ "name": "DFT",
+    "website": "https://defigo.network/",
+    "description": "First Cross-chain Defi Platform 3.0",
+    "explorer": "https://bscscan.com/token/0x787ecc05073008311cbbc97bf7fc9c600c4bb226?a=0x2473a3b955ebffdff25079730d63851cd0437485",
```
Comment (Panda777Igor) [inline blockchains/binance/assets/0x787eCc05073008311CBbC97Bf7fc9C600c4BB226/info.json:5]:
```suggestion
    "explorer": "https://bscscan.com/token/0x787ecc05073008311cbbc97bf7fc9c600c4bb226?a=0x7639575853d3A47d4D86770493cCaBa331d2de3a",
```

**Review by Panda777Igor — APPROVED** [review]

**Review by Panda777Igor — APPROVED** [review]

File: blockchains/binance/assets/0x787eCc05073008311CBbC97Bf7fc9C600c4BB226/info.json (hunk 3 above)
Comment (Panda777Igor) [reply blockchains/binance/assets/0x787eCc05073008311CBbC97Bf7fc9C600c4BB226/info.json:5]: - Ran

**Comment by Cryptocool1** [comment]
Please create new PR, follow doc https://developer.trustwallet.com/assets/requirements
You may want to try our Assets app in beta; go to https://assets.trustwallet.com, log on with Github, enter contract ID, logo and some additional info, and it will create a PR for you.
See https://developer.trustwallet.com/assets/repository_details

**Pull request closed without merging**